build.gradle.kts

```diff
@@ -54,10 +56,44 @@ kotlin {
     jvmToolchain(8)
 }
 
-publishing {
-    publications {
-        create<MavenPublication>("mavenJava") {
-            from(components["java"])
+mavenPublishing {
+    publishToMavenCentral(SonatypeHost.DEFAULT, true)
+    signAllPublications()
+
+    pom {
+        name = "faire-detekt-rules"
+        description = "Detekt rules for Faire"
+        inceptionYear = "2023"
```

Comment:
2024

Reply:
I checked commits and it was 2023 December. Or is that not considered the true inception 🙏

Reply:
ya oki 2023 it is